### ISSUE TITLE
lv_img_get_angle() returns incorrect number on negatively rotated images. 

### ISSUE THREAD

**Issue by vritzka** [kisvegabor/lvgl#4017]
- [x] Read the [FAQ](https://docs.lvgl.io/master/intro/index.html#faq)
- [x] Check the related part of the [Documentation](https://docs.lvgl.io/)
- [x] Update lvgl to the latest version
- [ ] Reproduce the issue in a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html)

### Describe the bug

We have an image 'myImage'.

_lv_img_get_angle(myImage)_ will return 0; 

Now we rotate the image left with _lv_img_set_angle(myImage, -200)._

When I now run _lv_img_get_angle(myImage)_, it returns 65336.

But the maximum possible rotation is 3600. Therefore it should return 3400 ([Documentation](https://docs.lvgl.io/master/widgets/img.html#_CPPv416lv_img_set_angleP8lv_obj_t7int16_t))

### To Reproduce

```c

// 1. Create any image
ui_needle1 = lv_img_create(ui_Home_Screen);
lv_img_set_src(myImage, &ui_img_1113527240);
lv_obj_set_width( myImage, LV_SIZE_CONTENT);  
lv_obj_set_height( myImage, LV_SIZE_CONTENT);   
lv_obj_set_x( myImage, 10 );
lv_obj_set_y( myImage, 10 );

// 2. Read angle of image

lv_img_get_angle(myImage);      // 0

// 3. Rotate the image negatively by -200

lv_img_set_angle(myImage, -200);

// 4 Read angle of image again:

lv_img_get_angle(myImage);      // 65336

```
-->

### Expected behavior
After rotating an image by -200 with lv_img_set_angle(), the value returned by lv_img_get_angle() should be 3400.

**Comment by kisvegabor** [comment]
Which LVGL version do you use?

**Comment by vritzka** [comment]
I’m using version 8.3.0

**Comment by kisvegabor** [comment]
It was already fixed in `master` but I've just fixed it in the `release/v8.3` branch too.
See e8d8f399e

**Comment by vritzka** [comment]
Thanks 😊
